src/components/SettlementButton.js

```diff
@@ -64,63 +57,11 @@ class SettlementButton extends React.Component {
             value: CONST.IOU.PAYMENT_TYPE.ELSEWHERE,
         });
 
-        // Venmo requires an async call to the native layer to determine availability and will be added as an option if available.
-        this.checkVenmoAvailabilityPromise = null;
-
         this.state = {
             buttonOptions,
         };
     }
 
-    componentDidMount() {
-        this.addVenmoPaymentOptionToMenu();
-    }
-
-    componentWillUnmount() {
-        if (!this.checkVenmoAvailabilityPromise) {
-            return;
-        }
-
-        this.checkVenmoAvailabilityPromise.cancel();
-        this.checkVenmoAvailabilityPromise = null;
-    }
-
-    /**
-     * @returns {Boolean}
-     */
-    doesRecipientHaveValidPhoneLogin() {
-        return this.props.recipientPhoneNumber && ValidationUtils.isValidUSPhone(this.props.recipientPhoneNumber);
-    }
-
-    /**
-     * Adds Venmo, if available, as the second option in the menu of payment options
-     */
-    addVenmoPaymentOptionToMenu() {
-        if (this.props.currency !== CONST.CURRENCY.USD || !this.doesRecipientHaveValidPhoneLogin()) {
-            return;
-        }
-
-        this.checkVenmoAvailabilityPromise = makeCancellablePromise(isAppInstalled('venmo'));
```

Comment:
Is `isAppInstalled` used anywhere else? I think we only added it for Venmo.

Reply:
Gonna have a look! Thanks!

Reply:
Removed!